types.ts

```diff
@@ -1010,12 +1004,20 @@ export enum ReservedAuthenticationNames {
   System = 'systemAuthentication',
 }
 
+export type AdminAuthenticationDefinition = AllowedAuthentication & {
+  /**
+   * If true, this authentication can be used to sync permissions associated with data
+   * in addition to the data itself.
+   */
+  canSyncPermissions?: boolean;
+};
+
 /**
  * TODO(patrick): Unhide this.
  * @hidden
  */
 export interface AdminAuthentication {
```

Comment:
why not just move it as a separate property here?

Reply:
That works too, but I took my approach because this property used to live in the nested authentication object and this change preserves that structure. It means that I don't need to change anything in the unforked Packs:

```
pack.addAdminAuthentication({
  name: 'AdminOAuth2Authentication',
  description: 'Requests permissions only available to Miro Company Admins',
  displayName: 'Admin Authentication',
  authentication: {...commonAuthenticationDef, scopes: adminAuthScopes, canSyncPermissions: true},
});
```

Reply:
interesting... i'm curious what @ekoleda-codaio thinks

Reply:
I was actually just working on documenting admin auth today, and I did find it a bit surprising that `canSyncPermissions` was in the nested auth definition and not in the outer `AdminAuthentication` definition. It would be a simpler interface IMO if it was at the outer level.

Reply:
I agree, I think it's worth the pack code churn to move up a level

Reply:
I also found that that only valid value was `true`, which sort of makes it feel like busy work. If there is any value in having this field if there is no variation in the value?

Reply:
the default value is `true`, so it shouldn't be busy work... but sure we could just remove it for now, until we introduce some concept of admin auths that do something different.

Reply:
Removed it in afb7cf4e4a7767aeb3388184182304ca5e800718